modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+
+// This is WITHOUT Kiali.
+// TP1 content. TP banner is persistent and handled by Tim O'Keefe in a different PR.
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
```

Comment:
```suggestion
You can integrate {SMProductName} with user-workload monitoring.
```

I guess with SEO searches, statements dependent on the lead=in sentence will get dropped, so its best to focus on the user perspective, IMO.